lib/src/main/java/com/thenewboston/data/dto/confirmationvalidatorapi/bankconfirmationservicesdto/BankConfirmationServices.kt

```diff
@@ -0,0 +1,32 @@
+package com.thenewboston.data.dto.confirmationvalidatorapi.bankconfirmationservicesdto
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class BankConfirmationServicesList(
+    @SerialName("count")
+    val count: Int,
+    @SerialName("next")
+    val next: String? = null,
+    @SerialName("previous")
+    val previous: String? = null,
+    @SerialName("results")
+    val services: List<BankConfirmationServices>
+)
+
+@Serializable
+data class BankConfirmationServices(
+    @SerialName("id")
+    val id: String,
+    @SerialName("created_date")
+    val createdDate: String,
+    @SerialName("modified_date")
+    val modifiedDate: String,
+    @SerialName("end")
+    val end: String,
+    @SerialName("start")
+    val start: String,
+    @SerialName("validator")
+    val bank: String
+)
```

Comment:
Let's try not to duplicate this model class. The only difference is the `bank` and `validator ` properties.

We can make them nullable.

For example:
@Serializable
data class ConfirmationServices(
    @SerialName("id")
    val id: String,
    @SerialName("created_date")
    val createdDate: String,
    @SerialName("modified_date")
    val modifiedDate: String,
    @SerialName("end")
    val end: String,
    @SerialName("start")
    val start: String,
    @SerialName("validator")
    val validator: String? = null
    @SerialName("bank")
    val bank: String? = null
)